Sources/MapboxMaps/Annotations/Generated/PolylineAnnotationManager.swift

```diff
@@ -243,6 +250,131 @@ public class PolylineAnnotationManager: AnnotationManagerInternal {
             delegate?.annotationManager(
                 self,
                 didDetectTappedAnnotations: tappedAnnotations)
+            var selectedAnnotationIds = tappedAnnotations.map(\.id)
+              let allAnnotations: [PolylineAnnotation] = self.annotations.map { annotation in
+              var mutableAnnotation = annotation
+                if selectedAnnotationIds.contains(annotation.id) {
+                    if mutableAnnotation.isSelected == false {
+                        mutableAnnotation.isSelected = true
+                    } else {
+                        mutableAnnotation.isSelected = false
+                    }
+                }
+                selectedAnnotationIds.append(mutableAnnotation.id)
+                return mutableAnnotation
+            }
+
+            self.annotations = allAnnotations
+
+        }
+    }
+
+    internal func createDragSourceAndLayer() {
+        var dragSource = GeoJSONSource()
+        dragSource.data = .empty
+        try? style.addSource(dragSource, id: "dragSource")
+
+        let dragLayerId = "drag-layer"
+        var dragLayer = LineLayer(id: dragLayerId)
+        dragLayer.source = "dragSource"
+        try? style.addLayer(dragLayer, layerPosition: .default)
+    }
+
+    internal func handleDragBegin(_ mapboxMap: MapboxMap, annotation: Annotation, position: CGPoint) {
+        createDragSourceAndLayer()
+
+        guard let annotation = annotation as? PolylineAnnotation else { return }
+
+        try? mapboxMap.style.updateLayer(withId: "drag-layer", type: LineLayer.self, update: { layer in
+            layer.lineColor = annotation.lineColor.map(Value.constant)
+            layer.lineOpacity = annotation.lineOpacity.map(Value.constant)
+            layer.lineWidth = annotation.lineWidth.map(Value.constant)
+            guard let linePattern = annotation.linePattern else { return }
+            layer.linePattern = Value.constant(ResolvedImage.name(linePattern))
+        })
+
+        self.annotationBeingDragged = annotation
+        self.annotations.removeAll(where: { $0.id == annotation.id })
```

Comment:
It seems like multiple ids are being caught by this logic: 

https://user-images.githubusercontent.com/7976026/198631042-d4bb5a2f-1683-4683-be5f-f24e5f1d0b46.mp4

Reply:
Those appear to all the same lines that cross in multiple locations. Can you add shorter lines and verify? I remember testing this example previously.